alpine/workfile.py

```diff
@@ -494,11 +498,18 @@ def wait_until_finished(self, process_id, verbose=False, query_time=10, timeout=
                         " It now has status '{2}'.".format(process_id, timeout, stop_status))
 
                 if verbose:
-                    print("\rWorkflow in progress for ~{0:.2f} seconds.".format(wait_total)),
+                    print("\rWorkflow in progress for ~{0:.1f} seconds.".format(wait_total)),
```

Comment:
I'd like to stick with printing since this method is specifically is for giving immediate feedback to users. How about always 

`self.logger.debug("\rWorkflow in progress for ~{0:.2f} seconds.".format(wait_total))`

and also printing when the user requests it?